lib/recorder.js

```diff
@@ -154,6 +155,16 @@ module.exports = {
     });
   },
 
+  saveFirstAsyncError(err) {
```

Comment:
I have some doubts that this should be a part of recorder...

Reply:
@DavertMik Can you please give some more information?

How do you think, where we should save first Async error?